crates/viewer/re_data_ui/src/instance_path.rs

```diff
@@ -524,16 +524,25 @@ fn preview_single_blob(
         .component_mono::<components::Blob>(blob_descr)?
         .ok()?;
 
+    // Media type comes typically alongside the blob.
     let media_type = find_and_deserialize_archetype_mono_component::<components::MediaType>(
         components,
         blob_descr.archetype_name,
     )
     .or_else(|| components::MediaType::guess_from_data(&blob));
 
-    let video_timestamp = find_and_deserialize_archetype_mono_component::<components::VideoTimestamp>(
```

Comment:
Why does `find_and_deserialize_archetype_mono_component` work for `MediaType` but not for `VideoTimestamp`?

Reply:
tried to capture this weird bit in those comments sounds like that worked only partially :)

`MediaType` is actually associated with _several_ archetypes, in fact we don't have a `Blob` archetype. Lemme make this a lil bit more clear there...